docs/multicluster/quick-start.md

```diff
@@ -47,8 +47,9 @@ mode, and all member clusters in a ClusterSet must use the same tunnel type.
 ## Steps with antctl
 
 `antctl` provides a couple of commands to facilitate deployment, configuration,
-and troubleshooting of Antrea Multi-cluster. This section describes the steps
-to deploy Antrea Multi-cluster and set up the example ClusterSet using `antctl`.
+and troubleshooting of Antrea Multi-cluster. Please refer to the [`antctl` installation guide](../antctl.md#installation)
```

Comment:
I feel better to move "Refer to antctl installation guide..." to the beginning of the next paragraph.

Reply:
Done